linode/manager
# refactor: [M3-6563] - Create util function to filter plan selecting t…

## Description 📝
 Common util function `getPlanSelectionsByPlanType` servers plan selections to Linode, Database and LKE plan tables.


## How to test 🧪

- Validate Linode Create / Clone / Update / resize / backup flows
- Check Plans table is rendering all plan types as expected.
- Validate LKE Plans table rendering all plan types as expected.
- Validate Database plans table rendering all plan types as expected.
- yarn test
- yarn cy:run
